modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-labels-and-tags_{context}"]
+= Configuring the user-defined labels and tags for GCP
+
+
+GCP provides tags that help to identify and organize the resources created for a specific {product-title} cluster, making them easier to manage. 
+
+You can define labels & tags on the GCP resources only during cluster creation that is available from {product-title} version {product-version}. User-defined labels and tags are not supported for {product-title} clusters upgraded to {product-title} version {product-version}.
+
+User-defined labels and {product-title} specific labels are applied only to resources created by {product-title} installer and its core operators such as `machine-api-provider-gcp`, `cluster-image-registry-operator`, `gcp-filestore-csi-driver-operator`, `gcp-pd-csi-driver-operator` and not on the resources created by any other operators or the kubernetes in-tree components that creates resources for example the ingress load balancers.
+
+User-defined tags are attached to resources created by {product-title} installer and its core operators such as `cluster-image-registry-operator` and not on the resources created by any other operators or the kubernetes in-tree components. In {product-title} version {product-version}, resources created by `machine-api-provider-gcp`, `gcp-filestore-csi-driver-operator`, `gcp-pd-csi-driver-operator` are not tagged.
+
+[NOTE]
+====
+Labels & tags once defined during the install-config creation phase cannot be modified or updated with new labels & tags after cluster creation.
+====
+
+Support for user-defined labels & tags for GCP resources is available in {product-title} version {product-version} as Technology Preview (TP).
+
+Service Account used for cluster creation must have the `TagUser` role assigned to it, required for tags defined at both organization and project level.
```

Comment:
FYI About the required role(s), I need to get a successful installation to double confirm. Will let you know then, thanks!

Reply:
No issues.

Reply:
@jianli-wei any updates?